sites/bcjobs/50-off-month-one.html

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: BC Jobs - 50% Off First Month
+landing-background-image: '/assets/img/handshake.jpg'
+permalink: /bcjobs/50-off-month-one
+post_url: "https://www.bcjobs.ca/coupons/50-off-month-one"
+region: "British Columbia"
+site_url: bcjobs.ca
+site_title: BC Jobs
+
+---
+
+{% include 	pages/50-off-month-one.html
+%}
```

Comment:
Strangely this one is the only one that's multi-line.

Reply:
Yes, that's leftover because I was passing some params in there, which format-wise look better on multi lines. I'll clean that up.

Good candidate for 'squash'?

Reply:
Either `squash` or new commit is fine.